pkg/reconciler/brokercell/broker_targets_config.go

```diff
@@ -133,6 +136,36 @@ func (r *Reconciler) addToConfig(ctx context.Context, b *brokerv1beta1.Broker, t
 	})
 }
 
+func targetsConfigMapsEqual(cm1, cm2 *corev1.ConfigMap) bool {
+	if !equality.Semantic.DeepEqual(cm1.Data, cm2.Data) {
+		return false
+	}
+	// The broker targets ConfigMap BinaryData holds the serialized TargetsConfig
+	// proto, and therefore cannot be safely compared with equality.Semantic.DeepEqual.
+	// Instead, use proto.Equal to compare protos.
+	if len(cm1.BinaryData) != len(cm2.BinaryData) {
+		return false
+	}
+	v1Proto := &config.TargetsConfig{}
```

Comment:
nit: would rename to something like proto1 to not confuse with proto versions.

Reply:
Done.